modules/seqexec/web/client/src/main/scala/seqexec/web/client/components/SeqexecUI.scala

```diff
@@ -118,6 +121,19 @@ object SeqexecUI {
     case _                            => s"Seqexec - ${site.show}"
   }
 
+  // Prism from url params to config page
+  def configPageP(instrumentNames: Map[String, Instrument]): Prism[(String, String, Int), SequenceConfigPage] = Prism[(String, String, Int), SequenceConfigPage] {
```

Comment:
it would be nice to simplify these but there is not at the moment a simple way to compose these in `Monocle`